DigitalExcellence/dex-frontend
# Recommendations on homepage

## Description

This PR shows project recommendations on the new home page.

## Type of change


## Checklist


## Steps to Test or Reproduce

If you can't run the backend in Docker I recommend to hardcode an array of projects in the recommendation service.
1. Hardcode return statement of recommendationservice.
2. Run backend (identity, API)
3. Run frontend, log in with any user.
4. Hardcoded recommendation should show up above "What is DeX".

To test this feature you should run all backend containers except the notificationsystem and jobscheduler.

1. Run backend
2. Log in with "bob", "bob"
3. Call /api/project/export endpoint to synchronize MSSQL with ElasticSearch
4. Make sure 2 different users have 1 similar like (like on same project) and 3 different likes (like on a project the other user doesn't like).
5. Go to home page logged in as one of the two users.
6. See project recommendations above "What is DeX".

If unclear, please contact me.

## Link to issue

Closes: #497
